internal/config/config.go

```diff
@@ -197,12 +197,11 @@ func (c *Config) SelectUser(index int) (*models.User, error) {
 
 // SelectDefaultUser returns the default user or nil if there is no default user
 func (c *Config) SelectDefaultUser() (*models.User, error) {
-	for _, user := range c.Users {
-		if user.Alias == constants.DefaultAlias {
-			return &user, nil
-		}
+	user, err := c.SelectUserByAlias("default")
```

Comment:
How about using `internal/constants.DefaultAlias` or deleting that file? DefaultAlias constant would be used here only.

Reply:
Good catch. I guess the "correct" way to do this is suing the constant value. If we use the constant in more places at some point in the future this is probably better maintainable. Let me know your thoughts about this.

Reply:
Okay great! Will update this.

Reply:
Resolved with 7fa366b